discovery-infra/test_infra/consts.py

```diff
@@ -41,6 +41,21 @@
 DEFAULT_NAMESPACE = 'assisted-installer'
 
 
+CNV_PARAMS = {
```

Comment:
Instead of each operator having its own constant, this can be changed to a nested dictionary:

```python
OPERATOR_PARAMS = {
	  "cnv: {
	      "worker_memory": 360,
	      "master_memory": 150,
	      "worker_vcpu": 2,
	      "master_vcpu": 4,
	  },
	  "ocs: {
	      "worker_memory": 14000,
	      "master_memory": 24000,
	      "worker_vcpu": 6,
	      "master_vcpu": 8,
	  }
}
```

Then the `_resource_param` methods can be simplified from a `for` + switch-like `if-else` chain to just:

```python
def _resource_param(value, param_name):
	try:
		return sum(OPERATOR_PARAMS[operator][param_name] 
				   for operator in utils.get_env("OLM_OPERATORS", "").lower().split())
	except KeyError as e:
		raise ValueError(f"Unknown operator name {e.args[0]}. Available operators are {', '.join(operator_params)}")
```

Reply:
Also please document where all those 360, 150, 2, 4 14000, 24000, 6, 8 constants are coming from - is there some official documentation they were extracted from? Trial and error?